jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_warning/diagnostics_rx_error_summarizer.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import csv
+import rospy
+import argparse
+from diagnostic_msgs.msg import DiagnosticArray
+
+
+class DiagnosticsRxErrorSummarizer():
+    def __init__(self, is_csv=False):
+        self.subscribe()
+        self.keys = ['timestamp']
+        self.count = 0
+        self.output_file = 'diagnostic_agg_rx_error.csv'
+        self.is_csv = is_csv
+
+    def subscribe(self):
+        self.sub = rospy.Subscriber("/diagnostics_agg",
+                                    DiagnosticArray,
+                                    self.callback,
+                                    queue_size=1)
+
+    def write_keys(self):
+        with open(self.output_file, 'w') as f:
+            writer = csv.writer(f)
+            writer.writerow(self.keys)
+
+    def write_values(self):
+        with open(self.output_file, 'a') as f:
+            writer = csv.writer(f)
+            writer.writerow(self.values)
+
+    def callback(self, msg):
+        status = msg.status
+        self.values = [msg.header.stamp.secs]
+        if self.is_csv:
+            print(self.count)
+            for s in status:
+                if 'values' not in dir(s):
+                    continue
+                for v in s.values:
+                    if 'RX Error' in v.key:
+                        if self.count == 0:
+                            self.keys.append(s.name + ' - ' + v.key)
+                            self.values.append(v.value)
```

Comment:
I think only timestamps are written into csv now.
```suggestion
                        self.values.append(v.value)
```
I would appreciate it if the timestamp is displayed on stdout, too.